documentation/blog/2024-04-01-whats-new-in-svelte-april-2024.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "What's new in Svelte: April 2024"
+description: 'Svelte Summit Spring on April 27! Plus: Maps, Dates and Sets'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Spring is just around the corner (for half the globe at least), which means [Svelte Summit Spring](https://www.sveltesummit.com/) is coming soon! The event will take place on April 27 and could feature you (yes you!). The call for proposals is happening now so submit a talk (or three!) at [this form](https://www.sveltesummit.com/2024/spring/submit).
+
+Also, this month, a ton of new features have been merged into Svelte 5 to make it the best version of Svelte yet. We'll get into them all below, so let's jump right in!
+
+
+## What's new in Svelte
+
+[Svelte 5 is in preview](https://svelte-5-preview.vercel.app/docs/introduction) and gets closer to release every day. Below, you'll find some highlights from its [changelog](https://github.com/sveltejs/svelte/blob/main/packages/svelte/CHANGELOG.md):
+
+- state/derived/props can be explicitly exported from components again (**5.0.0-next.62**, [#10523](https://github.com/sveltejs/svelte/pull/10523))
+- `bind:value` now allows for a dynamic `type` attribute - fixing issues with common input bindings (**5.0.0-next.66**, [#10608](https://github.com/sveltejs/svelte/pull/10608))
+- A new reactive `Date` object simplifies time and date reactivity within Svelte (**5.0.0-next.67**, [Docs](https://svelte-5-preview.vercel.app/docs/runes#state-frozen-reactive-map-set-and-date), [#10622](https://github.com/sveltejs/svelte/pull/10622))
+- `SSR` HTML mismatch validation has been improved and provides clearer error messages (5.0.0-next.69, [#10658](https://github.com/sveltejs/svelte/pull/10658))
+- **Breaking:** Slots inside templates with a `shadowrootmode` attribute are now preserved (**5.0.0-next.73**, [#10721](https://github.com/sveltejs/svelte/pull/10721))
+- **Breaking:** The `$props()` no longer accepts a generic type argument. Instead, component authors should type their props like any other variable declaration (**5.0.0-next.76**, [#10694](https://github.com/sveltejs/svelte/pull/10694))
+- A new reactive `Map` object acts just like a normal Map but it will keep reactivity for enumeration, and all its API methods (**5.0.0-next.79**, [Docs](https://svelte-5-preview.vercel.app/docs/runes#state-frozen-reactive-map-set-and-date), [#10803](https://github.com/sveltejs/svelte/pull/10803))
+- The webkitdirectory DOM boolean attribute is now supported (**5.0.0-next.81**, [#10847](https://github.com/sveltejs/svelte/pull/10847))
+- Form resets are now taken into account for two way bindings (**5.0.0-next.82**, [Docs](https://svelte-5-preview.vercel.app/docs/breaking-changes#other-breaking-changes-bindings-now-react-to-form-resets), [#10617](https://github.com/sveltejs/svelte/pull/10617))
+
```

Comment:
Maybe include $bindable? https://github.com/sveltejs/svelte/pull/10851

Reply:
Also there's a reactive Set too.

Reply:
Weird. I don't see bindable in the release notes. Added to this list anyway.